.devcontainer/Dockerfile

```diff
@@ -37,8 +37,8 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN pip3 install --upgrade pip
 
 # Install AWS CLI v2
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
-    unzip awscliv2.zip && \
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip" && \
```

Comment:
Another way to do this is with `TARGETARCH` instead of shelling out: https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Reply:
Are we able to update these `uname` uses with `TARGETARCH` instead?

Reply:
Unfortunately we can't swap in `TARGETARCH` here because awscli uses `aarch64` and `x86_64` in the URL instead of `arm64` and `amd64`. However I can check the value of `TARGETARCH` and set an env var instead of shelling out if that is preferred.